src/server_manager/cloud/digitalocean_api.ts

```diff
@@ -251,3 +251,15 @@ function makeValidDropletName(name: string): string {
   // Remove all characters outside of A-Z, a-z, 0-9 and '-'.
   return name.replace(/[^A-Za-z0-9\-]/g, '');
 }
+
+export async function digitalOceanRetry<T>(
```

Comment:
It is with the recursive call, but based on our brainstorming, this will change!

My plan is to throw a domain event from the Account (ManagedServerRepository) and handle or rethrow in the individual apps (DigitalOceanCreateServerApp). Atm, I can't differentiate between a network error and an authorization error, but I will try @alalamav 's suggestion and check out the CORS middleware layer we added for DigitalOcean.